src/locales/messages.fr-FR.xlf

```diff
@@ -39,7 +39,7 @@
         <target>Page non trouvée</target>
         <context-group purpose="location">
           <context context-type="sourcefile">src/app/app-routing.module.ts</context>
-          <context context-type="linenumber">32</context>
+          <context context-type="linenumber">37</context>
```

Comment:
So when modify the code, the translation line is modified too.
You have a tool update it or you do this manually?